### ISSUE TITLE
Filter by confidence level wrong when filtered by "low confidence"

### ISSUE THREAD

**Issue by MJWainwright** [epistemonikos/isoq#87]
Hi Damian,

A user noticed that when they filtered by "low confidence" in the iSoQ table, both the "low confidence" and the "very low confidence" findings are displayed. I was able to reproduce this error on the test site and shared the project with you

![image](https://user-images.githubusercontent.com/72507850/107567911-37528a00-6bde-11eb-9fef-8c82a49545f1.png)

**Comment by MJWainwright** [comment]
@damian-garrido Any update on this ticket?

**Comment by MJWainwright** [comment]
@damian-garrido Any update on this one?

**Comment by MJWainwright** [comment]
@damian-garrido I've just checked this again the bug is still there. I added 4 findings - 1 high confidence, 1 moderate confidence, 1 low confidence, and 1 very low confidence. The filter for high, moderate and very low work. BUT, there is an error in the filter for "low confidence", when you filter for low confidence it also displays "very low confidence" findings in the table.

![image](https://user-images.githubusercontent.com/72507850/129245984-3e3ef658-1db3-4d9a-b3b6-cafef37baf46.png)
